src/hooks/useRecoveryInfo.ts

```diff
@@ -0,0 +1,42 @@
+"use client";
+
+import { SocialRecoveryModule } from "abstractionkit";
+import { useAccount, usePublicClient } from "wagmi";
+import { Address } from "viem";
+import { useQuery } from "@tanstack/react-query";
+import { socialRecoveryModuleAbi } from "@/utils/abis/socialRecoveryModuleAbi";
+
+// Track a recovery process that has been approved by guardians
+export function useRecoveryInfo(safeAddress?: Address) {
```

Comment:
should we use a more descriptive name then?
```suggestion
export function useApprovedOngoingRecoveryInfo(safeAddress?: Address) {
```